app/models/cart.rb

```diff
@@ -238,4 +217,22 @@ def fields_for_display
       EXCLUDE_FIELDS_FROM_DISPLAY.include? key}.map{ |key,value,label|
       [label, value] }
   end
+
+  def client
+    self.origin
+  end
```

Comment:
Origin doesn't exist on the ncr model. Are you saying you want the interface method renamed to `origin`?